### ISSUE TITLE
Project built with Vue TS component fails

### ISSUE THREAD

**Issue by JesusCrow** [akxcv/vuera#48]
I was using Vuetify components, but recently they updated them to support TS.
My build completes, but website breaks with an error:
![image](https://user-images.githubusercontent.com/6717694/42375454-4184bacc-8124-11e8-8ec0-68a6bced2d20.png)
Project (Vue based) bricks like this while using any component from Vuetify (not a single React component included).
I think that Vuera is trying to do something with Vue TS components, because it thinks that it's React component.

**Comment by JesusCrow** [comment]
This is "simplest" reproduction I could make:

```bash
git clone https://github.com/JesusCrow/vuera-vuetify-repro
cd vuera-vuetify-repro
yarn && yarn bug
# open browser and it's console
```
Sorry for this poor repro (it only installs vuera, vuetify, ands single component from vuetify and builds it with vue-cli). Need to replace vue-cli for building with webpack only. Using other library (Vuetify) to crash the build, should replace it with standalone self written TS component.

I thought it might have something to do with the way libs/components are installed.
https://github.com/vuetifyjs/vuetify/issues/4068
It also might be that isReactComponent cooperator fails? Because I'm not using even a single React component. I'm currently away, but I'll try to test it out next week.

**Comment by akxcv** [comment]
Hi, sorry for taking a long time. I've poked at the issue a bit and found that installing vuera's `VuePlugin` _after_ Vuetify seems to work.
My guess is that Vuetify is registering its components globally at startup, and somewhere there's a clash. Further investigation required.

**Comment by JesusCrow** [comment]
Hmm, changing installation order did not help in my current project.

**Comment by akxcv** [comment]
The weird thing is that the error only occurs in the minified code. I have a feeling that the problem occurs when using vuera not with TS, but with Vuetify...

**Comment by stlbucket** [comment]
i am experiencing the same issue and am using Vuetify

**Comment by stlbucket** [comment]
everything works fine during dev, but when i build and run from /dist, i get the same error as @JesusCrow

**Comment by stlbucket** [comment]
https://github.com/akxcv/vuera/issues/46

are they related?

**Comment by stlbucket** [comment]
https://github.com/akxcv/vuera/issues/46

are they related

**Comment by stlbucket** [comment]
i've traced it to here (i think).  looks like it is failing on my v-app, which is the top-level wrapper for vuetify.

![image](https://user-images.githubusercontent.com/2467892/47375962-5abc3880-d6a6-11e8-9d56-ec66930dd926.png)


![image](https://user-images.githubusercontent.com/2467892/47375920-3d876a00-d6a6-11e8-8d06-b4ed6d9d78a4.png)

maybe this function needs another filter?

![image](https://user-images.githubusercontent.com/2467892/47376084-9951f300-d6a6-11e8-84f4-84bf3748a197.png)

**Comment by stlbucket** [comment]
btw, that is in:   **vuera.cjs.js**

**Comment by stlbucket** [comment]
this actually "fixed" the issue, but it doesn't seem like the correct answer to me.

i'll not file a PR unless you want that @akxcv.  i'm hoping you know a better way?


![image](https://user-images.githubusercontent.com/2467892/47383527-b93ee200-d6b9-11e8-9830-bac4e7e430f9.png)

**Comment by akxcv** [comment]
Yes, this function might not cover all cases correctly. The truth is, I don't know for sure how to 100% accurately distinguish a React component from a Vue component. The current function seemed to work, but it surely might fail in some weird cases (especially since Vue allows you to do so many different things).

I'd be very happy if you could submit a PR! I don't think that the current solution is quite what we're looking for, but if you could nail this down, it would be perfect. Unfortunately, I don't have a lot of time to debug such cases, and I don't know all the possible cases, too...

**Comment by akxcv** [comment]
Should be fixed via #71
